data/js/src/main/scala/sigma/data/js/Isos.scala

```diff
@@ -0,0 +1,152 @@
+package sigma.data.js
+
+import org.ergoplatform.{ErgoBox, ErgoBoxCandidate}
+import org.ergoplatform.ErgoBox._
+import scorex.crypto.authds.ADKey
+import scorex.util.ModifierId
+import scorex.util.encode.Base16
+import sigma.{Coll, Colls}
+import sigmastate.fleetSdkCommon.distEsmTypesBoxesMod.{Box => FBox}
+import sigmastate.fleetSdkCommon.distEsmTypesCommonMod.{Amount, HexString}
+import sigmastate.fleetSdkCommon.distEsmTypesRegistersMod.NonMandatoryRegisters
+import sigmastate.fleetSdkCommon.{distEsmTypesBoxesMod => boxesMod, distEsmTypesCommonMod => commonMod, distEsmTypesRegistersMod => registersMod, distEsmTypesTokenMod => tokenMod}
+import sigma.Extensions._
+import sigma.ast.syntax.GroupElementConstant
+import sigma.ast.{Constant, GroupElementConstant, SType}
+import sigma.data.Iso.isoStringToArray
+import sigma.data.{CGroupElement, Digest32Coll, Iso}
+import sigma.js.GroupElement
+import sigma.serialization.{ErgoTreeSerializer, ValueSerializer}
+
+import scala.scalajs.js
+
+/** Definitions of isomorphisms for sigma-data module. */
```

Comment:
This ScalaDoc can be avoided by just renaming Isos to Isomorphisms.

What are isomorphisms? What are they used for ?

Reply:
> What are isomorphisms? What are they used for ?

Very convenient abstraction to define conversions between different representations of the same data. See `Iso` trait, it is explained there. 
Also this https://en.wikipedia.org/wiki/Isomorphism

Improved the related ScalaDocs.